python/ray/_private/worker.py

```diff
@@ -2343,29 +2343,38 @@ def connect(
 
     if mode == SCRIPT_MODE:
         # Add the directory containing the script that is running to the Python
-        # paths of the workers. Also add the current directory. Note that this
-        # assumes that the directory structures on the machines in the clusters
-        # are the same.
-        # When using an interactive shell, there is no script directory.
-        # We also want to skip adding script directory when running from dashboard.
-        code_paths = []
-        if not interactive_mode and not (
-            namespace and namespace == ray_constants.RAY_INTERNAL_DASHBOARD_NAMESPACE
+        # paths of the workers. Also add the current directory.
+        #
+        # Note that this is only meaningful when all of these are true:
```

Comment:
can you also mention why here briefly (to the code comment)?

Reply:
done